doc/user_guide/mva.rst

```diff
@@ -104,8 +104,16 @@ the total variance specified, and the components above this value will be
 styled distinctly from the remaining components to show which are considered
 signal, as opposed to noise. Alternatively, by providing an integer value
 for ``threshold``, the line will be drawn at the specified component (see
-below).  The "elbow" position can be auto-estimated and a vertical line drawn to
-represent the no of signicant pca components by specifying "vline" as True. 
+below).  The no of significant components can be estimated and a vertical line drawn to
+represent this by specifying "vline" as True. In this case the elbow or knee is found 
```

Comment:
"In this case the elbow" --> "In this case, the elbow"

Reply:
fixed